plugins/hfp_ag_bluez5.c

```diff
@@ -129,8 +173,18 @@ static DBusMessage *profile_new_connection(DBusConnection *conn,
 						g_strdup(device), g_free);
 	g_io_channel_unref(io);
 
-	g_hash_table_insert(connection_hash, g_strdup(device),
-						GINT_TO_POINTER(fd_dup));
+	dev = g_new0(struct device_info, 1);
+	dev->fd = fd_dup;
+
+	dev->card = ofono_handsfree_card_create(0, OFONO_HANDSFREE_CARD_TYPE_GATEWAY,
+											NULL, NULL);
```

Comment:
Lines > 80 chars here. There are a few more instances of this around, please correct.